promscale/installation/promscale-with-timescale-cloud/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Install Promscale Connector with Timescale Cloud
+excerpt: Install Promscale with Timescale cloud on Kubernetes, Docker, virtual machine, or bare metal
+product: promscale
+keywords: [analytics]
+tags: [learn, prometheus]
+related_pages:
+  - /promscale/:currentVersion:/guides/resource-recomm/
+  - /promscale/:currentVersion:/send-data/
+---
+
+import Install from "versionContent/_partials/_cloud-installation.mdx";
+import CreateServicePromscale from "versionContent/_partials/_cloud-create-service-promscale.mdx";
+
+# Install Promscale with Timescale cloud
+
+You can install Promscale with Timescale Cloud by running Promscale Connector on
+and leverage the Timescale Cloud as the storage layer for your observability data.
+To get started with Promscale with Timescale Cloud first create a Timescale Cloud
+service and then install Promscale Connector.
+
+## Install Timescale Cloud
+
+Timescale Cloud is a hosted, cloud-native TimescaleDB service that allows you to
+quickly spin up new TimescaleDB instances. You can
+[try Timescale Cloud for free][sign-up], no credit card required.
+
+Powered by [TimescaleDB][timescale-features], Timescale Cloud is an innovative
+and cost-effective way to store and analyze your time-series data. Get started
+super fast with demo data, or your own dataset, and enjoy the security of
+automated upgrades and backups.
```

Comment:
This is old wording. You can grab the partial from here instead: _partials/_cloud-intro.mdx

Reply:
done!!!